src/components/__tests__/InstantSearch.js

```diff
@@ -27,6 +28,9 @@ it('passes props to InstantSearch.js', () => {
       stalledSearchDelay: 250,
       searchFunction,
     },
+    slots: {
+      default: '',
+    },
```

Comment:
not needed anymore after the fix with &&, no?

Reply:
https://github.com/algolia/vue-instantsearch/pull/1014/commits/5df22838e95fe5890e420a6bc98edd0d3b9fb4db